### ISSUE TITLE
Can't follow @bsky.brid.gy@bsky.brid.gy and the profile does not show up at all

### ISSUE THREAD

**Issue by AlarithUhde** [snarfed/bridgy-fed#1429]
Good evening/morning/day!

I cannot use the bridge with my account. Possibly this is related to [this issue](https://github.com/snarfed/bridgy-fed/issues/1093), but in my case I cannot see even the profile (with the profile picture) as in the other issue. Also, another Mastodon user [reported a problem](https://hci.social/@ebuie/113006317147139248) a while ago that might be identical but I am not sure.

What happens?
- when I try to click on @bsky.brid.gy@bsky.brid.gy as linked in someone else's post, all I get is a new tab with the following content (details may vary, but it looks like this):

![Screenshot_20240825_120429](https://github.com/user-attachments/assets/427b5c3e-fa9a-4006-8970-a56bb924748a)

- This happens with firefox and chrome on the desktop (Debian 12), and with Tusky (opens firefox) on android (9).
- I also tried to search for the profile in the Mastodon search, but it does not show up.
- Same if I open a new tab and enter this URL: https://bsky.brid.gy/bsky.brid.gy

It seems like some of these variants work for other people (see Mastodon thread linked above), but not for me and apparently not for the thread OP either.

[My account](hci.social/@alarith) is on hci.social and it seems to work for other users on that server.

So by now it does not seem to be a problem with the server or any of the apps/browsers specifically. I ran out of explanations why it won't work.

Thank you for the consideration and best wishes!

**Comment by Tamschi** [comment]
There are no other accounts bridged from `hci.social` according to [this directory page](https://blue.mackuba.eu/directory/fedi).
Just to make sure, are `bsky.brid.gy` or `fed.brid.gy` listed among the 'Moderated servers' on https://hci.social/about? (I can't view this list without an account.)

---

Have you tried to search for `https://bsky.brid.gy/bsky.brid.gy` in the Mastodon search box? (Press enter afterwards.)
_While logged-in_, this should normally bring up the account as single result within the Mastodon web interface.

Note that while logged-out in the web interface (like when viewing another instance after selecting 'Open original page'), the search seems to not show the result even if it's known there. Tapping the @-mention in these cases also opens the raw JSON like in the screenshot above, rather than loading the profile into Mastodon.

(We could do better and show a simplified profile page when someone visits the page with a browser, but I think that's fairly low priority compared to ironing out the core functions for now.)

Sorry if these are steps you've already taken. Mastodon's search can be finicky &ndash; you need to search for `@bsky.brid.gy@bsky.brid.gy` *exactly* (and press enter) sometimes &ndash; so I'd like to make sure this isn't the reason.

---

Also, while it doesn't quite match the symptoms you describe, have you ever blocked the bridge?

**Comment by snarfed** [comment]
Merging into #1009

**Comment by snarfed** [comment]
Duplicate of #1009

**Comment by AlarithUhde** [comment]
Thank you for the detailed response. I asked the hci.social admins and they confirmed that neither `bsky.brid.gy` nor `fed.brid.gy` are on the list of moderated servers.

I tried these steps exactly as described (searched for both https://bsky.brid.gy/bsky.brid.gy and @bsky.brid.gy@bsky.brid.gy in the search bar while logged in) and several other ways (e.g., clicking on the profile link from someone else's post). When I tried it just now, there is no way to follow, but the json page was replaced with another one. It seems to redirect to https://fed.brid.gy/ now.

So: Servers are not blocked, typical method does not work, not sure how I could resolve this.

**Comment by snarfed** [comment]
@AlarithUhde ugh, sorry. Confusing! Sadly I'm running out of ideas on my end for this too, I don't understand what your Mastodon instance is doing here.

**Comment by jfietkau** [comment]
Should we try to consolidate the diagnosis here, can someone from the hci.social team with admin access join this thread?

I initially thought I wouldn't be able to do much to help, until I remembered that I have the account for the directory bot on hci.social. 😀 With that, I have confirmed that the bridge bot account cannot be pulled up via the search box (neither by handle nor by ID), and other known bridged accounts can also not be viewed. Also, mentions of @bsky.brid.gy@bsky.brid.gy (and, as it happens, @snarfed.org@snarfed.org) in posts coming from another Mastodon server do not work, and are instead treated as external web links. For clarity: this means it has nothing to do with how Bridgy Fed itself outputs mentions, since the post is coming from another Mastodon server.

I believe this behavior is consistent with an administrative server block on the bridge (and on snarfed.org).

My suggestions for diagnosis from the hci.social side:

* Go to Preferences -> Moderation -> Federation, enter "bsky.brid.gy" into the search filter, and double-check that there are no domain-level blocks listed. Check for "brid.gy" as well.
* Go to Preferences -> Moderation -> Accounts, set location to "Remote", domain to "bsky.brid.gy" and search. Does anything at all come up?
